Source/CarthageKit/Xcode.swift

```diff
@@ -1254,7 +1254,7 @@ public func buildInDirectory(directoryURL: NSURL, withConfiguration configuratio
 	return SignalProducer { observer, disposable in
 		// Use SignalProducer.buffer() to avoid enumerating the given directory
 		// multiple times.
-		let (locatorBuffer, locatorObserver) = SignalProducer<(ProjectLocator, [String]), CarthageError>.buffer()
+		let (locatorBuffer, locatorObserver) = SignalProducer<(ProjectLocator, [String]), CarthageError>.buffer(Int.max)
 
 		buildableSchemesInDirectory(directoryURL, withConfiguration: configuration, forPlatforms: platforms)
```

Comment:
> Is that function covered by tests though? This might not be exactly equivalent since it was previously eagerly started, so we should make sure it doesn't break anything.

I think this should be covered by the [XcodeSpec](https://github.com/Carthage/Carthage/blob/f0124fc1aa9c3537169d6a5ba6a0c552d61f6563/Source/CarthageKitTests/XcodeSpec.swift#L106-L112). :wink: